src/uct/ib/rc/base/rc_iface.h

```diff
@@ -276,10 +281,10 @@ struct uct_rc_iface_send_op {
     uint16_t                      sn;
     uint16_t                      flags;
     unsigned                      length;
+    uct_rc_iface_t                *iface;
```

Comment:
maybe better pass iface as parameter to completion functions rather than adding a field?

Reply:
not sure, this would affect almost all rc ops. do you think it is worth it?

Reply:
is it possible to test perf diff?

Reply:
so do you think add iface as param would be worse perf, or just wanted to avoid changing too much code?